### ISSUE TITLE
Make reference list multilingual-capable

### ISSUE THREAD

**Issue by asmecher** [pkp/pkp-lib#5303]
The submission's reference list field is currently not multilingual.

In cases where publishing happens in multiple character sets (e.g. Cyrillic and Latin), having a list for each language is a frequently-requested option.

See https://forum.pkp.sfu.ca/t/how-in-ojs-3-1-2-0-in-a-standard-theme-to-make-input-of-literature-for-each-language/57124/12 for discussion, including a link to the JATS orthodoxy on the subject.

**Comment by ipoliarush** [comment]
Does the platform currently not support bibliography with support for multiple languages?

**Comment by asmecher** [comment]
No, not at the moment. See the linked discussion for some of the issues involved.

**Comment by fgnievinski** [comment]
related discussion:
https://github.com/pkp/pkp-lib/discussions/8207#discussioncomment-3468892

**Comment by jyhein** [comment]
I have started working on this. There may be two (or more) options how to do this using the forms:

1. Add one reference (all languages; separate fields for different languages) at a time using an add button. Store primary language to 'citations' table and additional languages to 'citation_settings' table. 'citation_id' links additional languages to the primary. Needs more UI changes than option 2.

2. Add references using a text box per language. Store all languages to 'citations' table. Add 'locale' field to 'citations' table. References are not linked between languages but UI stays pretty much the same.

**Comment by asmecher** [comment]
@bozana, I wanted to get your thoughts on this. My thoughts are...
- Even if it's not important for data consumers now, we shouldn't permit mismatched reference lists to proliferate because they'll be hard to clean up later. So we shouldn't allow someone to paste e.g. a 21-reference list in English, and a 15-reference list in Spanish. (It is possible we could clean it up later by e.g. matching DOIs, but it smells like a headache with a lot of outliers.)
- We probably should decide on a single way to store references. Currently we have both the whole-list field, and individual citations split by line. There are still some remnants of the old citation assistant, e.g. the `citation_settings` table.

**Comment by bozana** [comment]
@asmecher, yes... let me think about it... I will come back here...

**Comment by asmecher** [comment]
I think @GaziYucel's work on citation management will solve this request in a much better way with OJS/OMP/OPS 3.5.0. Closing as a WONTFIX.

**Comment by GaziYucel** [comment]
For interested, please follow this issue for multilingual and structured citations. 

https://github.com/pkp/pkp-lib/issues/10692
